### DISCUSSION TITLE
Close port 53 after use

### DISCUSSION THREAD

**Discussion by MarthinHauge** [hassio-addons/addon-adguard-home#269] [General]
Hi. After using this adguard addon, my port 53 is open. How can i close it again? 

Thanks

**Comment by frenck** [comment]
? I'm not sure about your question. This add-on is a DNS server, it will use the port when the add-on is in use.

**Reply by MarthinHauge** [reply]
okey. When i nmap myself after turning off add-on it show that my 53 port is still open

**Reply by frenck** [reply]
nmap what? As in, port 53 is used on multiple interface by different tools and programs. Have you checked which process is using that port on your system?
